### PR TITLE
Update styles for contact form

### PR DESCRIPTION
**Pull request by hectoruch**
Add style for form-contact.

### CODE REVIEW

**Comment by hectoruch** [comment]
There are some sass-lint errors using input-placeholder and appearence rules on scss files, but there are not other way to do that.

File: app/src/components/ContactUs/ContactUsForm.jsx [hunk 1]
```diff
@@ -96,25 +100,18 @@ class ContactUsForm extends Component {
           </select>
         </div>
 
-        <label htmlFor="subject">Subject</label>
-        <input
+        <label htmlFor="subject">Message</label>
+        <textarea
           type="text" id="contact_subject"
-          placeholder="subject"
```
Comment (tiagojsag) [inline app/src/components/ContactUs/ContactUsForm.jsx]: @hectoruch don't remove or rename fields without a good reason - these are required.

**Comment by tiagojsag** [comment]
@hectoruch the "thank you" page is too short. On my screen (15' 1080p) I see a huge white bar bellow the footer. This has to go. I also added

File: app/styles/components/c-contact-form.scss [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    color: $color-4;
+    font-size: $font-size-normal;
+    margin: 50px auto 100px;
+    max-width: 250px;
+    padding: 20px 0;
+    text-transform: uppercase;
+    width: 100%;
+  }
+
+  ::-webkit-input-placeholder {
```
Comment (andresgnlez) [inline app/styles/components/c-contact-form.scss]: These trhee selectors can be in one statement. Example:

`.selectorA,
.selectorB,
.selector C {}`

**Comment by tiagojsag** [comment]
Also, at least on my browser, the different options inside the <select> show up in the same shade of gray as the placeholder, and not in black like the content of the other fields

File: app/styles/components/c-contact.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
+@import '../settings';
+
+.c-contact {
+  background: $color-7 url('../../../assets/images/home_slider_1-1.png') center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: initial;
+
+  .layer-cover {
```
Comment (andresgnlez) [inline app/styles/components/c-contact.scss]: Is this kind of a veil on the image? If that, use pseudo elements `before` and `after` and you don't need to add unnecessary DOM.  If you have any question, ask me.

File: app/styles/components/c-contact-form.scss [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    background-color: $color-4;
+    border: solid 1px $color-10;
+    border-radius: 0;
+    height: 200px;
+    margin-bottom: 10px;
+    outline: 0;
+    padding: 10px;
+  }
+
+  .submit-contact {
```
Comment (andresgnlez) [inline app/styles/components/c-contact-form.scss]: We have this button styles in other parts of the site: that's clearly a component. Use `c-button` component to set all components you find similar and set as many modifiers (`-red`, `-big`, `-whatever`) you need.

This way, you don't have to repeat button styles every time you find a button.

File: app/styles/components/c-contact-form.scss (hunk 2 above)
Comment (clementprdhomme) [inline app/styles/components/c-contact-form.scss]: Nope, they can't. If we do so, some of them are skipped (don't know why).

**Comment by clementprdhomme** [comment]
Is it normal the background image is so pixelated? On FF the blue overlay doesn't reach the bottom of the background.

**Comment by tiagojsag** [comment]
Images are not final yet

File: app/src/components/ContactUs/ContactUsForm.jsx [hunk 5]
```diff
@@ -96,25 +100,28 @@ class ContactUsForm extends Component {
...
           required
           onChange={this.handleChange}
         />
 
-        <label htmlFor="description">description</label>
-        <input
-          type="textarea"
+        <label htmlFor="description">Message *</label>
+        <textarea
+          type="text"
```
Comment (clementprdhomme) [inline app/src/components/ContactUs/ContactUsForm.jsx]: This attribute doesn't exist for `textarea`.

**Comment by hectoruch** [comment]
Update styles:

- Thank You page error fix.
- Update cover layer styles.
- Add new class on button file -> on new PR I will mix the styles.
- Delete type="text" on textarea
- Create classes for inputs on form.

**Comment by tiagojsag** [comment]
the select options should be black, and only the placeholder should be gray

**Comment by hectoruch** [comment]
@tiagojsag 
Me and @andresgnlez are trying to fix it.

**Comment by tiagojsag** [comment]
Select still has the same problem (FF and chromium)
on FF, the required fields have a red border

**Comment by clementprdhomme** [comment]
@tiagojsag  It seems the red border on Firefox is due to default styles the browser adds on required fields.

@hectoruch You can just reset the `box-shadow` property to remove them.

**Comment by hectoruch** [comment]
Update:

- Border red fix on firefox browser.
- Option color error fix.

**Comment by tiagojsag** [comment]
The select is improving, but still not there yet ;) please check the static pages in /public/forms to see how clement solved it, and shamelessly copy his code :P

File: app/src/components/ContactUs/ContactUsForm.jsx [hunk 6]
```diff
@@ -1,12 +1,15 @@
...
+import buttonStyle from '../../../styles/components/c-button.scss';
 
 class ContactUsForm extends Component {
   constructor(props) {
     super(props);
     this.state = {
       submitted: false,
-      showFormResponse: false
+      showFormResponse: false,
+      colorSelect: '#b8b8b8',
```
Comment (tiagojsag) [inline app/src/components/ContactUs/ContactUsForm.jsx]: not the biggest fan of this here

File: app/styles/components/shared/c-header.scss [hunk 7]
```diff
@@ -8,6 +8,7 @@
   height: 95px;
   justify-content: space-between;
   padding: 0 30px;
+  position: relative;
```
Comment (tiagojsag) [inline app/styles/components/shared/c-header.scss]: are we sure this does not affect any other page?

Comment (andresgnlez) [inline app/styles/components/shared/c-header.scss]: He says it doesn't affect.

**Comment by andresgnlez** [comment]
The `textarea` has an unnecessary margin in the end, can you remove it?

**Comment by andresgnlez** [comment]
And please, rebase from`develop` to have some `_base` styles like `wrap` and add it to surround the form.

**Pull request merged**